app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
```

Comment:
1. bỏ ngoặc `provide(` này đi nhé
2. Sử dụng I18n nhé
Check toàn bộ các file tương tự nhé